database/seeders/PublishStartDateSeeder.php

```diff
@@ -11,11 +11,14 @@ class PublishStartDateSeeder extends Seeder
     private $tableNames = [
         'articles',
         'events',
-        'selections',
+        'highlights',
     ];
 
     public function run(): void
     {
+        // XXX I think I was mistaken, the migration this came from:
+        // https://github.com/art-institute-of-chicago/artic.edu/pull/5/files#diff-7870073f898417541722fbad31dd4e28f0d780f7c9b28a1b8773ca2852067665
+        // looks like it's renaming/recasting the published column. nikhil, is that correct?
```

Comment:
That's right. The models here used to only have a published toggle, but this migration was part of work to allow content creators to seta  future publish date and have content automatically publish at that time. See more here: https://art-institute-of-chicago.atlassian.net/browse/WEB-674

So this line says at the time the `publish_start_date` was added, take all published content and give that column a value of `now()` so it will be considered published when dates are looked at.